options/locale/locale_en-US.ini

```diff
@@ -2511,7 +2510,7 @@ first_page = First
 last_page = Last
 total = Total: %d
 
-dashboard.new_version_hint = Gitea %s is now available, you are running %s. Check the <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">blog</a> for more details.
+dashboard.new_version_hint = Gitea %s is now available, you are running %s. Check <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">blog</a> for more details.
```

Comment:
No.
```suggestion
dashboard.new_version_hint = Gitea %s is now available, you are running %s. Check the <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">blog</a> for more details.
```

Reply:
You need to change something to trigger the re-translation. Please provide another writing style.